docs/user-guides/workspaces-shared.md

```diff
@@ -0,0 +1,62 @@
+# Shared Workspaces
+
+Multiple users can securely connect to a single Coder workspace for debugging, programming, and code review. 
```

Comment:
I'm not sure code review is really a use case people will want from shared workspaces